src/components/MoviesListErrorBoundary/index.jsx

```diff
@@ -0,0 +1,17 @@
+import React, { Fragment } from 'react';
+import { movies } from '../MoviesList/model';
+
+const MoviesListErrorBoundary = (props) => {
```

Comment:
You need also to define two lifecycle methods here getDerivedStateFromError and componentDidCatch for Error Boundary because now it doesn't catch error from children components.
[https://reactjs.org/docs/error-boundaries.html#gatsby-focus-wrapper](https://reactjs.org/docs/error-boundaries.html#gatsby-focus-wrapper)